nucypher/cli/commands/alice.py

```diff
@@ -500,14 +508,12 @@ def grant(general_config,
         label = click.prompt(f'Enter label to grant Bob {bob_verifying_key[:8]}', type=click.STRING)
 
     if not force and not expiration:
-        if ALICE.duration_periods:
-            # TODO: use a default in days or periods?
-            expiration = maya.now() + timedelta(days=ALICE.duration_periods)  # default
-            if not click.confirm(f'Use default policy duration (expires {expiration})?'):
-                expiration = click.prompt('Enter policy expiration datetime', type=click.DateTime())
-        else:
-            # No policy duration default default available; Go interactive
-            expiration = click.prompt('Enter policy expiration datetime', type=click.DateTime())
+        expiration_prompt = 'Enter policy expiration (Y-M-D H:M:S)'
```

Comment:
I know we allowed `click.DateTime()` before which allowed for formats `["%Y-%m-%d", "%Y-%m-%dT%H:%M:%S", "%Y-%m-%d %H:%M:%S"]`, but can we actually do anything more granular than day i.e. hours, minutes, seconds? If we can't, then we should explicitly specify the format `"%Y-%m-%d"` to click.DateTime, and only prompt for `Y-m-d` to be specified.

Reply:
Yes.  If find these precise format requirements to be very frictional.

Reply:
> can we actually do anything more granular than day i.e. hours, minutes, seconds?

Yes, epoch seconds is used at the contract-level.

Reply:
Interesting. I guess in that case there is payment for periods and any part thereof i.e. payment for fees is period granular (effectively UTC day), but enaction of a policy is second granularity (@vzotova do I have this right?).

Reply:
Policy rates are calculated on a per-period basis, but retain their original timeatamps:

https://github.com/nucypher/nucypher/blob/main/nucypher/blockchain/eth/sol/source/contracts/PolicyManager.sol#L239